src/System.Windows.Forms/src/System/Windows/Forms/ListBox.AccessibleObject.cs

```diff
@@ -37,19 +35,30 @@ public ListBoxAccessibleObject(ListBox owningListBox) : base(owningListBox)
         // We need to provide a unique ID. Others are implementing this in the same manner. First item is static - 0x2a (RuntimeIDFirstItem).
         // Second item can be anything, but it's good to supply HWND.
         internal override int[] RuntimeId
-            => new int[]
+        {
+            get
             {
-                RuntimeIDFirstItem,
-                PARAM.ToInt(_owningListBox.InternalHandle),
-                _owningListBox.GetHashCode()
-            };
+                this.TryGetOwnerAs(out ListBox? owner);
+                if (owner is null)
```

Comment:
Nit: I also suggest to simplify it to the following
```suggestion
                if (!this.TryGetOwnerAs(out ListBox? owner))
```
Same in few other places here.